scripts/deploy-sdks.ts

```diff
@@ -78,6 +78,27 @@ type Answers = {
     process.exit(1)
   }
 
+  let tag = 'latest'
+  const prerelease = semver.prerelease(version)
+  if (prerelease) {
+    tag = (prerelease[0] + '').trim()
```

Comment:
the type of `prerelease` is `Array<string | number>`, because it splits the suffix of the version, eg `1.1.1-alpha.0` would be `['alpha', 0]`. I, as a developper, know that the first element will be a string, but typescript doesn't. I could have done `prerelease[0] as string` too.